tests/cxx/direct_near.h

```diff
@@ -0,0 +1,29 @@
+//===--- direct_near.h - test input file for iwyu -------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// This file is the same as tests/direct.h, only intended to be used without
+// provided header search path -I .
+
+#ifndef DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_TESTS_DIRECT_NEAR_H_
+#define DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_TESTS_DIRECT_NEAR_H_
+
+#include "indirect.h"
+
+#endif  // DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_TESTS_DIRECT_NEAR_H_
+
+/**** IWYU_SUMMARY
```

Comment:
No need for an IWYU summary pattern here, it's only evaluated for associated headers and headers named by `--check_also`